recipes/slycot/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "slycot" %}
+{% set version = "0.2.0" %}
+{% set md5 = "35c1737180ecee67ba6574d65dde85b1" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 200
+  # Conda forge lacks openblas on Windows.
+  skip: true  # [win]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - toolchain
+    - gcc          # [osx]
+    - python
+    - blas 1.1 {{ variant }}
+    - libgfortran  # [linux]
+    - openblas 0.2.19|0.2.19.*
+    - numpy x.x
+    - setuptools
```

Comment:
From the following comment on setup.py, it looks like it's using distutils. I'm asking because if setuptools is being used you should add `--single-version-externally-managed --record record.txt`
```
 # Use setuptools for these commands (they don't work well or at all
# with distutils).  For normal builds use distutils.
```